core/src/main/c/share/jit/x86.h

```diff
@@ -38,36 +38,39 @@ namespace questdb::x86 {
         return {Mem(vars_ptr, 8 * idx, type_size), type, data_kind_t::kMemory};
     }
 
+    // Reads length of variable size column with header stored in data vector (string, binary).
     jit_value_t read_mem_varlen(Compiler &c,
                                 uint32_t header_size,
                                 int32_t column_idx,
-                                const Gp &column_address,
-                                const Gp &varlen_indexes_ptr,
+                                const Gp &cols_ptr,
+                                const Gp &varsize_indexes_ptr,
                                 const Gp &input_index) {
-        // Column has variable-length data. Load the value of its header
-        // (i.e., data length). It can also indicate a NULL value, encoded as length -1.
-        // We reach the header by looking up its offset in the varlen index.
-        // First try avoiding the data-dependent load of the header: load the next entry
-        // from the varlen index and subtract from it header_size + the value of the
-        // current one. This is equal to the length of the value.
+        // Column has variable-size data with header stored in data vector.
+        // First, we load this and the next data vector offsets from the aux vector.
```

Comment:
Consistent naming cont'd -- "data vector" and "column" are two names for the same thing; also "varsize index" and "aux vector".